index.html

```diff
@@ -502,7 +508,7 @@ <h2><dfn>PerformanceObserverInit</dfn> dictionary</h2>
           <dl>
             <dt><dfn>buffered</dfn></dt>
             <dd>A flag to indicate whether buffered entries should be queued
-            into observer's buffer.</dd>
+            into observer's buffer for each entry.</dd>
```

Comment:
Not sure what this addition is trying to clarify.

Reply:
Like below, at some point I was thinking of the observers buffer differently. Removed.